src/components/LineContent/index.tsx

```diff
@@ -53,11 +54,15 @@ export default class LineContent extends Component<any, any> {
         }
 
         return (
-            <span className={lineContent} style={style}>
-                {data &&
-                    data.map((part, n) => (
-                        <LinePart part={part} format={formatPart} key={`line-${number}-${n}`} />
-                    ))}
+            <span className={lineContent} style={style} onMouseUp={onMouseUp}>
+                {data?.map((part, n) => (
```

Comment:
This was originally 
```suggestion
                {data && data.map((part, n) => (
```

While it looks like the default is an empty array, I think the original implementation is a bit safer.